sdcm/cluster.py

```diff
@@ -1274,6 +1274,10 @@ def wait_manager_server_up(self, verbose=True, timeout=300, port=None):
             wait.wait_for(func=self.is_manager_server_up, port=self.OLD_MANAGER_PORT,
                           step=10, text=text, timeout=timeout, throw_exc=True)
 
+    def restart_manager_server(self, port=None):
```

Comment:
shouldn't the port have a default, then calls to it won't have to "remember" it?